paper-input.html

```diff
@@ -221,7 +223,9 @@
 
   <template id="v1">
     <!-- Need to bind maxlength so that the paper-input-char-counter works correctly -->
-    <iron-input bind-value="{{value}}" id="input" slot="input"
+    <iron-input bind-value="{{value}}" slot="input"
+        class="input-element"
+        id$="[[_inputId]]"
```

Comment:
should this be done in the native input below?

Reply:
no, i fyou look, the `id` was in the `iron-input` previously

Reply:
Chrome complains about native inputs having the same id, e.g. this is the simplified DOM when we're in shadyDOM mode, and Chrome complains about double `#nativeInput`:
```html
  <form>

    <label for="p0">password p0</label>
    <iron-input id="p0">
      <input type="password" id="nativeInput">  
    </iron-input>

    <label for="p1">password p1</label>
    <iron-input id="p1">
      <input type="password" id="nativeInput">  
    </iron-input>

  </form>
```
http://jsbin.com/gelevij/1/edit?html,console,output

Here the repro with paper-input having the same issue:
```html
  <form>

    <paper-input type="password" label="password p0"></paper-input>

    <paper-input type="password" label="password p1"></paper-input>
    
  </form>
```
http://jsbin.com/puyiwiw/1/edit?html
http://output.jsbin.com/puyiwiw/?wc-ce=true&wc-shadydom=true

Reply:
oh i see (yes: the error in Chrome is what we're trying to fix btw). I guess `nativeInput` also needs a separate ID (paper-input doesn't know if you're stamping a 1.x or 2.x template, so that's why things have to be named similarly)

Reply:
I think we only need the fix for `nativeInput`, not `<iron-input>`

Reply:
Sure, but `inputElement` the getter has to point to the right thing (either the native input in 1.x or the `iron-input` in 2.x, and since it's the same code, it also has to have a correctly generated name (so that the getter works).

Reply:
I can just remove the `nativeInput` id here, tho, since it's only used in one place

Reply:
Nice call! 👍